src/lwc/geFormField/geFormField.js

```diff
@@ -45,11 +53,16 @@ export default class GeFormField extends LightningElement {
         if(this.isRichText) {
             this.checkRichTextValidity();
         }
+
+        if(this.sourceFieldAPIName === DI_DONATION_AMOUNT.fieldApiName) {
+            // fire event for reactive widget component containing the Data Import field API name and Value
+            // currently only used for the Donation Amount.
+            fireEvent(null, 'widgetData', { Donation_Amount__c: this.value });
```

Comment:
Yeah, that would make sense here. We can call this one whatever we want to though it never gets converted into or from a proper sObject. It's a variable name on a global object that a widget can use to figure out what the entered amount is on the donation.

Reply:
This line of code isn't affected by namespaces and I think changing it to  `{ donationAmount: this.value }` would be good since Donation_Amount__c is only a variable name/key. Having it named the same as a SFDC Custom Field API Name is kinda misleading.